components/brave_ads/core/internal/common/strings/string_conversions_util.cc

```diff
@@ -18,4 +22,30 @@ std::string BoolToString(const bool value) {
   return value ? kTrue : kFalse;
 }
 
+std::vector<float> ConvertStringToVector(const std::string& string,
+                                         const std::string& delimiter) {
+  const std::vector<std::string> vector_string = base::SplitString(
+      string, delimiter, base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+  std::vector<float> vector;
+  vector.reserve(vector_string.size());
+  for (const auto& element_string : vector_string) {
+    double element;
+    base::StringToDouble(element_string, &element);
+    vector.push_back(static_cast<float>(element));
+  }
+
+  return vector;
+}
+
+std::string ConvertVectorToString(const std::vector<float>& vector,
+                                  const std::string& delimiter) {
+  std::vector<std::string> string_vector;
+  string_vector.reserve(vector.size());
+  for (const auto& element : vector) {
```

Comment:
nit: for consistency in ads would be great if we could use `item` instead of `element`